scripts/sendText.js

```diff
@@ -10,9 +10,12 @@ function sendText(text){
 
     console.log('cursor position:',getCurrentCursorPosition('input'));
 
+    var baseUrlEndpoint = "http://localhost:5000/process?"
+
     //Third, we use the fetch method to call the localhost asynchronously (Synchronously was not recommended)
     //This is where we get the results from the index.js file
-    fetch("http://localhost:8080/spellCheck?lng="+languageSelected+"&text="+value, {
+    //fetch("http://localhost:8080/spellCheck?lng="+languageSelected+"&text="+value, {
+    fetch( baseUrlEndpoint+ "lng=" + languageSelected+"&phrase="+value, {
```

Comment:
Did we change the code to fetch the data text  without database
Or only by reading the text files, we uploaded?

Reply:
Pream,
Per project requirements, we need to develop a text editor. Therefore, we are not handling a file submission but we are handling user input and sending it to the Flask endpoint created by kimaya.
Let me know if you have questions.
Thank you,
Julian